react/features/participants-pane/components/web/MeetingParticipants.tsx

```diff
@@ -112,6 +112,12 @@ function MeetingParticipants({
 
     return (
         <>
+            <span
+                aria-level = { 1 }
+                className = 'sr-only'
+                role = 'heading'>
+                { t('participantsPane.title') }
+            </span>
```

Comment:
changed the logic here after the rebase. Since https://github.com/jitsi/jitsi-meet/commit/50c4748d403415f58385f09f74d7ce5ec1128db2 added a potential visitors count in addition to the participants count, we can't rely on the participants count sentence to be the heading anymore. If we did, the visitors count would certainly be missed by people navigating from heading to heading.